{ 'fall_position_right': { 'head': [ { 'duration': 1.0,
                                       'name': u'fall_position_right',
                                       'positions': [0.0, 0.0012],
                                       'starttime': 0.0}],
                           'left_arm': [ { 'duration': 1.0,
                                           'name': u'fall_position_side_up',
                                           'positions': [ -0.8934,
                                                          -0.6375,
                                                          -0.1192,
                                                          0.0477,
                                                          -0.5361,
                                                          0.3916,
                                                          -0.0044],
                                           'starttime': 0.0}],
                           'left_leg': [ { 'duration': 1.0,
                                           'name': u'fall_position_side_up',
                                           'positions': [ 0.053,
                                                          -0.193,
                                                          0.1109,
                                                          0.9757,
                                                          0.9927,
                                                          -0.0366],
                                           'starttime': 0.0}],
                           'right_arm': [ { 'duration': 1.0,
                                            'name': u'fall_position_side_down',
                                            'positions': [ 2.4335,
                                                           0.7778,
                                                           -0.2246,
                                                           0.2017,
                                                           -1.5431,
                                                           -4.8103,
                                                           0.0283],
                                            'starttime': 0.0}],
                           'right_leg': [ { 'duration': 1.0,
                                            'name': u'fall_position_side_down',
                                            'positions': [ 0.0557,
                                                           -0.2319,
                                                           -0.1478,
                                                           -0.8672,
                                                           -0.9055,
                                                           -0.2244],
                                            'starttime': 0.0}],
                           'torso': [ { 'duration': 1.0,
                                        'name': u'fall_position_side_right',
                                        'positions': [0.1737, 0.5869],
                                        'starttime': 0.0}]}}
